services/121-service/test/program/update-program.test.ts

```diff
@@ -52,31 +51,6 @@ describe('Update program', () => {
     expect(updateProgramResponse.body.budget).toBe(program.budget);
   });
 
-  // ##TODO Enable/Refacetor/Remove this test after FSP config endpoints are implemented (and the logic is reused in update program). This test should than check if it is possible to add FSP config to a program if that is not aldreaddy covered by some other test
-  it.skip('should add an fsp to a program', async () => {
```

Comment:
is the new logic now covered in another test?

Reply:
We removed the functionality to also update program-financial-service-provider-configurations via PATCH /program. Added it to the handover [task](https://dev.azure.com/redcrossnl/121%20Platform/_sprints/taskboard/121%20Development%20Team/121%20Platform/Sprint%20141?workitem=30191)